app/logic/Calendar/Participant.ts

```diff
@@ -4,12 +4,23 @@ import { notifyChangedProperty } from "../util/Observable";
 
 export class Participant extends PersonUID {
   @notifyChangedProperty
-  response: InvitationResponse;
+  _response: InvitationResponse;
+  /** DTSTAMP when the status was sent that is captured in this object.
+   * Used during auto-update to avoid overwriting with older info. */
+  lastUpdateTime: Date | null;
```

Comment:
I have a vague memory of someone complaining because my names didn't match the RFC, which calls this a timestamp (the RFC does also have a last modified time, which is a separate thing).